client/grpc/oracle/proof/multi_store.go

```diff
@@ -9,54 +9,55 @@ import (
 // MultiStoreProof stores a compact of other Cosmos-SDK modules' storage hash in multistore to
 // compute (in combination with oracle store hash) Tendermint's application state hash at a given block.
 //
-//	                                           ________________[AppHash]_________________
-//	                                          /                                          \
-//	                      _________________[I14]_________________                     __[I15]__
-//	                     /                                        \				 /          \
-//	          _______[I12]______                          _______[I13]________    [G]         [H]
-//	         /                  \                        /                    \
-//	    __[I8]__             __[I9]__                __[I10]__              __[I11]__
+//	                                           ________________[AppHash]_________________________
+//	                                          /                                                  \
+//	                      _________________[N15]_________________                             __[N16]___
```

Comment:
Why need to change from I -> N

Reply:
because there is an 'I' used in the leaf node